File: 2016/round2/22/j.py
# UESTC Summer Training Petrozavodsk Summer-2015. Moscow Subregional 2010.
# Problem J, by Abreto.

line = raw_input();

nums = line.split(' ');

x = float(nums[0]);
y = float(nums[1]);

print x,y;

if x > 1:
    print int(x / y);
else:
    lx = len(nums[0]);
    if lx < 12:
        print int(x/y);
    else:
        print x;

File: 2017/r1/01/j.c
/* UESTC 2017 Summer Training #1 Div.2. Problem J, by Abreto<m@abreto.net>. */
#include <stdio.h>

int main(void)
{
    int i = 0;
    int N = 0, K = 0;
    int nd = 0;
    int ld = 0, cd = 0;

    scanf("%d%d", &N, &K);
    scanf("%d", &ld);
    for(i = 1;i < N;++i)
    {
        scanf("%d", &cd);
        if(ld != cd) nd++;
        ld = cd;
    }
    printf("%.8lf\n", (double)(K-1)*(double)nd/(double)K);

    return 0;
}
